pulsar/core/sippy/SipTransactionManager.py

```diff
@@ -22,27 +22,28 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA.
 
-from Timeout import Timeout
-from Udp_server import Udp_server
-from SipHeader import SipHeader
-from SipResponse import SipResponse
-from SipRequest import SipRequest
-from SipAddress import SipAddress
-from SipRoute import SipRoute
-from SipHeader import SipHeader
+from .Timeout import Timeout
+from .Udp_server import Udp_server
+from .SipHeader import SipHeader
+from .SipResponse import SipResponse
+from .SipRequest import SipRequest
+from .SipAddress import SipAddress
+from .SipRoute import SipRoute
+from .SipHeader import SipHeader
 from datetime import datetime
 from hashlib import md5
 from traceback import print_exc
 from time import time
 import sys, socket
+from functools import reduce
```

Comment:
Why is this introduced?

Reply:
`reduce` is [no longer a builtin in Python 3](https://docs.python.org/3.0/whatsnew/3.0.html#builtins):

> Removed reduce(). Use functools.reduce() if you really need it; however, 99 percent of the time an explicit for loop is more readable.